### PR TITLE
Add basic scene and controls

### LINKED ISSUE #1: Basic scene and controls
As an initial foundation, create a 2D world with a square character which can move around.

### PR DESCRIPTION
**Pull request by JacobALundgren**
**  Behavioral change: Introduce a single world screen with a player
square. Enable WASD keyboard control of player.

Issues: [#1]

### CODE REVIEW

**Comment by JacobALundgren** [comment]
Closes #1

**Comment by JacobALundgren** [comment]
Did a force push as I stupidly forgot to run the formatter.

**Comment by queleok** [comment]
It looks like there's an implicit dependency on either precompiled `shaderc` library, or `cmake`. I don't have either of those, so I got this error upon an attempt to run the package:
```
warning: shaderc: searching for native shaderc libraries on system;  use '--features build-from-source' to force building from source code
warning: shaderc: cannot find native shaderc library on system; falling back to build from source

error: failed to run custom build command for `shaderc-sys v0.7.3`

Caused by:
  process didn't exit successfully: `/Users/queleok/projects/rpg-game/target/debug/build/shaderc-sys-864c44d34bbbb17d/build-script-build` (exit status: 101)
  --- stdout
  cargo:warning=shaderc: searching for native shaderc libraries on system;  use '--features build-from-source' to force building from source code
  cargo:warning=shaderc: cannot find native shaderc library on system; falling back to build from source

  --- stderr
  thread 'main' panicked at '

  couldn't find required command: "cmake"

  ', /Users/queleok/.cargo/registry/src/github.com-1ecc6299db9ec823/shaderc-sys-0.7.3/build/cmd_finder.rs:55:13
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rpg-system-2d"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bevy = { version = "0.5.0", features = ["dynamic"] }
```
Comment (queleok) [inline Cargo.toml:9]: How about adding a build dependency (not so sure about the exact version, just added mine)?
```
[build-dependencies]
cmake = 3.21.2
```

**Review by queleok — CHANGES_REQUESTED** [review]
Please, check the suggestion for `Cargo.toml`

**Comment by JacobALundgren** [comment]
As discussed elsewhere, this seems to be [an issue in Bevy](https://github.com/bevyengine/bevy/issues/1360). Added a note suggesting the installation of cmake or the Vulkan SDK

**Review by queleok — APPROVED** [review]
Thank you, everything looks great :)

**Pull request merged**